script/releases/v0.5.2-rewardsv2/1-eoa.s.sol

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import {EOADeployer} from "zeus-templates/templates/EOADeployer.sol";
+import {RewardsCoordinator} from "src/contracts/core/RewardsCoordinator.sol";
+import {IDelegationManager} from "src/contracts/interfaces/IDelegationManager.sol";
+import {DelegationManager} from "src/contracts/core/DelegationManager.sol";
+import {StrategyManager} from "src/contracts/core/StrategyManager.sol";
+import {EigenLabsUpgrade} from "../EigenLabsUpgrade.s.sol";
+import {Test, console} from "forge-std/Test.sol";
+import {IPauserRegistry} from "src/contracts/interfaces/IPauserRegistry.sol";
+
+contract Deploy is EOADeployer {
```

Comment:
identical to 0.5.2 but sets `REWARDS_COORDINATOR_CALCULATION_INTERVAL_SECONDS` = 1 day